spec/controllers/idv/finance_controller_spec.rb

```diff
@@ -14,10 +14,12 @@
 
   describe '#create' do
     before do
+      user = sign_in_as_user
```

Comment:
If you rebase against master, you can use the new stub I added to avoid creating a user in the DB:
```ruby
user = User.new
stub_sign_in_before_2fa(user)
```